server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -93,22 +93,30 @@ export default class PrintLicenceRoutes {
   }
 
   getPdfFooter = (licence: Licence): string => {
+    let printVersion = licence.licenceVersion
+    const majorVersion = licence.licenceVersion.split('.')[0]
+    if (parseInt(majorVersion, 10) > 1) {
```

Comment:
variations have versions like 2.0, 3.0 etc. For these we just want to display the major version in the licence pdf like 2, 3, etc.